src/main/java/ru/ayaz/CommandExecutor.java

```diff
@@ -26,41 +21,49 @@ void executeCommand(UserMessage commandMessage) {
 
         switch (getCommand(text)) {
             case "#quit":
-                executeQuitCommand(commandMessage);
+                executeQuitCommand();
                 break;
             case "#ignore":
                 executeIgnoreCommand(commandMessage);
                 break;
             default:
                 UserMessage message = new UserMessage(nickname, "Unknown command");
-                broadcaster.sendMessageToOnlyOne(message, nickname);
+                userSocketHandler.sendMessage(message);
                 break;
         }
     }
 
-
-    private void executeQuitCommand(UserMessage commandMessage) {
-        UserSocketHandler userSocketHandler = userSocketHandlerMap.get(commandMessage.getSenderName());
+    private void executeQuitCommand() {
         try {
             userSocketHandler.closeStreams();
-            userMap.remove(commandMessage.getSenderName());
-            userSocketHandlerMap.remove(commandMessage.getSenderName());
         } catch (IOException e) {
             e.printStackTrace();
         }
     }
 
-
     private void executeIgnoreCommand(UserMessage commandMessage) {
         String nickname = commandMessage.getSenderName();
         String text = commandMessage.getText();
 
-        User user = userMap.get(nickname);
         try {
             user.ignoreUser(getFirstArgument(text));
         } catch (InvalidUserCommandException e) {
             UserMessage message = new UserMessage(nickname, "Invalid command");
-            broadcaster.sendMessageToOnlyOne(message, nickname);
+            userSocketHandler.sendMessage(message);
         }
     }
+
+    static String getCommand(String message) {
```

Comment:
First question which comes to mind is how to test private method with Junit?) 
I thought that while using TDD you write all methods as public and then you can refactor them into smaller, private methods. It means that we shouldnt test private methods directly.